README.md

```diff
@@ -8,32 +8,36 @@ Client side assets for the Baltimore County Public Library website
 - General Information - https://github.com/baltimorecounty/BCPL-assets/wiki
 - Developer Guide - https://github.com/baltimorecounty/BCPL-assets/wiki/Developer-Guide
 
-## Building Files for Distribution
-
-### Setup
+## Setup
 
 You will need to do these steps the first time you setup the project.
 
 1. [Clone this repository](https://help.github.com/articles/working-with-repositories/)
-1. Install [Git](https://git-scm.com/downloads) 
+1. Install [Git](https://git-scm.com/downloads)
 1. Install [Node](https://nodejs.org/download/)
 1. Install [Python 2.7.x](https://www.python.org/)
 1. Install [gulp](https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md)
 1. Install [C++ build tools](https://www.npmjs.com/package/windows-build-tools) (Must be installed as admin)
-1. Open terminal or command prompt to the root of your local repository
-1. Run `npm install` to install all local dependencies
+1. Install all local dependencies with:
+
+### `npm install`
+
+## Build
 
-### Build
+### `gulp`
 
-1. Open terminal or command prompt to the root of your local repository
-1. Run the `gulp` command
+Builds the app for dev and or production to the `dist` folder. It does the following:
 
-#### This process above will do the following.
+Note: When building for production ensure any constants files are pointing to production values.
 
 1. Compile all pug files into HTML, and copy into the 'dist' folder
-1. Compile and minfiy all sass stylesheets to css and place them in the 'dist/css' folder
+1. Compile and minify all sass stylesheets to css and place them in the 'dist/css' folder
```

Comment:
Hahaha I couldn't tell what changed here. Even with the word highlighted my brain refused to accept this.